README.md

```diff
@@ -28,47 +30,68 @@ Key elements of this approach include:
 7. Defining a clear, user-friendly standard for simulation design.
 
 ## Installation
+Simple installation via pip is available for users, developers should follow
+the directions under [Developer Installation](#developer-installation) below.
+
 A user-installation is achieved simply with `pip install pyuvsim`, or to get the
 bleeding-edge: `pip install https://github.com/RadioAstronomySoftwareGroup/pyuvsim`.
-This will install all dependencies.
+This will install all dependencies except pyradiosky, which is under active
+development and does not yet have a pip package on PyPI. To get pyradiosky, run
+`pip install https://github.com/RadioAstronomySoftwareGroup/pyradiosky`.
 
-By default, `mpi` capabilities are not enabled -- many of the utilities provided in
-`pyuvsim` do not require it. To use the simulator within `pyuvsim`, you should install
-`pyuvsim` with  `pip install pyuvsim[sim]`.
+By default, `mpi` capabilities are not enabled -- many of the utilities provided
+in `pyuvsim` do not require it. To use the simulator within `pyuvsim`, you
+should install `pyuvsim` with  `pip install pyuvsim[sim]`. Note that the
+`pyuvsim` simulator is intended to run on clusters running the linux operating
+system, but we do test against Mac OSX as well.
 
 There are a few more optional dependencies for `pyuvsim` which enable some features,
 such as `line_profiler` to use the built-in profiling, and `h5py` to write to HDF5
 file format. If you would like these tools as well as the full simulator, install
 `pyuvsim` with `pip install pyuvsim[all]`
 
-If you wish to manage dependencies manually, or are developing `pyuvsim` yourself, read
-on.
-
-Note that `pyuvsim` is intended to run on clusters running the linux operating system.
-
+If you wish to manage dependencies manually read on.
 
 ### Dependencies
 If you are using `conda` to manage your environment, you may wish to install the
-following before installing `pyuvsim`:
-
-    conda install -c conda-forge "numpy>=1.15" "astropy>=3.0" "scipy>1.0.1" "mpi4py>=3.0.0" "pyyaml>=5.1" "pyuvdata>=1.3.7"
-
-### Developing
-If you are developing `pyuvsim`, it is preferred that you do so in a fresh `conda`
-environment. The following commands will install all relevant development packages:
-
-    $ git clone https://github.com/RadioAstronomySoftwareGroup/pyuvsim.git
-    $ cd pyuvsim
-    $ conda create -n pyuvsim python=3
-    $ conda activate pyuvsim
-    $ conda env update -n pyuvsim -f environment.yml
-    $ pip install -e .
-
-This will install extra dependencies required for testing/development as well as the
-standard ones.
-
-The second-to-last line may also be replaced by `pip install -r requirements.txt` if you
-do not care about using `conda`.
+following packages before installing `pyuvsim`:
```

Comment:
Hmmm, so we've traded having this explicitly listed in `requirements.txt` for having them explicitly listed here? This still seems redundant. Can we just point people to the `setup.py` here?

Reply:
To be fair, I didn't add them here, they were already here. I just turned them into a bulleted list rather than an inline list. It can be hard to maintain separate lists, but I also think clarity in the readme is helpful.

Reply:
Ah, fair. If you're happy to have the maintenance overhead, then that's fine.